.env.test.example

```
@@ -50,46 +54,48 @@ QSTASH_NEXT_SIGNING_KEY=test-qstash-next
 # ---- Travel APIs -----------------------------------------------------------
-DUFFEL_ACCESS_TOKEN=test-duffel-token
-DUFFEL_API_KEY=test-duffel-key
-EPS_API_KEY=test-eps-key
-EPS_API_SECRET=test-eps-secret
+DUFFEL_ACCESS_TOKEN=
+DUFFEL_API_KEY=
+EPS_API_KEY=
+EPS_API_SECRET=
 EPS_BASE_URL=https://api.ean.com/2/rapid
 EPS_DEFAULT_CUSTOMER_IP=0.0.0.0
 EPS_DEFAULT_USER_AGENT=TripSage/1.0 (+https://tripsage.ai)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)